<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>miyao</class>
 <widget class="QWidget" name="miyao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>352</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>588</width>
    <height>352</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>588</width>
    <height>352</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_save_path">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>350</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>100</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;保存路径：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;生成密钥对&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_choose">
   <property name="geometry">
    <rect>
     <x>479</x>
     <y>100</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_generate">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>生成</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;从私钥生成公钥&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_choose_public">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>350</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_genrate_public">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>生成公钥</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_choose_public">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>250</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>选择私钥</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_bit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>50</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>4096</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8192</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2048</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1024</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;密钥位数：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
